Misc/NEWS.d/next/Library/2022-05-26-08-41-34.gh-issue-93243.uw6x5z.rst

```diff
@@ -0,0 +1 @@
+The :mod:`smtpd` dead battery was removed as slated by :pep:`594`.
```

Comment:
Slightly more neutral wording, feel free to keep "dead battery" if it tickles your fancy!
```suggestion
The :mod:`smtpd` module was removed per the schedule in :pep:`594`.
```